frontend/src/app/topic-drafts/convert-topic-draft-to-team/convert-topic-draft-to-team-dialogue-component/convert-topic-draft-to-team-dialogue.component.ts

```diff
@@ -4,21 +4,23 @@ import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
 import { TranslateService } from '@ngx-translate/core';
 import { Observable } from 'rxjs';
 import { map, switchMap, take, tap } from 'rxjs/operators';
-import { OkrDepartment } from '../../shared/model/ui/OrganizationalUnit/okr-department';
-import { OkrTopicDescription } from '../../shared/model/ui/OrganizationalUnit/okr-topic-description';
-import { OkrTopicDraft } from '../../shared/model/ui/OrganizationalUnit/okr-topic-draft/okr-topic-draft';
-import { Structure } from '../../shared/model/ui/OrganizationalUnit/structure';
-import { DepartmentMapper } from '../../shared/services/mapper/department.mapper';
-import { StructureMapper } from '../../shared/services/mapper/structure.mapper';
-import { TopicDescriptionMapper } from '../../shared/services/mapper/topic-description-mapper';
-import { SubmittedTopicDraftDetailsFormData } from '../submitted-topic-draft-details/submitted-topic-draft-details.component';
+import { OkrDepartment } from '../../../shared/model/ui/OrganizationalUnit/okr-department';
+import { OkrTopicDescription } from '../../../shared/model/ui/OrganizationalUnit/okr-topic-description';
+import { OkrTopicDraft } from '../../../shared/model/ui/OrganizationalUnit/okr-topic-draft/okr-topic-draft';
+import { Structure } from '../../../shared/model/ui/OrganizationalUnit/structure';
+import { DepartmentMapper } from '../../../shared/services/mapper/department.mapper';
+import { StructureMapper } from '../../../shared/services/mapper/structure.mapper';
+import { TopicDescriptionMapper } from '../../../shared/services/mapper/topic-description-mapper';
+import {
+  SubmittedTopicDraftDetailsFormData
+} from '../../topic-draft-details-dialogue-component/topic-draft-details-dialogue.component';
```

Comment:
Was ist heir für eine Änderung? ist die Formatierung in Zeile 14-16 so notwendig?